vmm/src/lib.rs

```diff
@@ -489,9 +489,10 @@ impl Vmm {
         &mut self,
         desired_vcpus: Option<u8>,
         desired_ram: Option<u64>,
+        desired_ram_w_balloon: Option<u64>,
```

Comment:
I don't really understand why this is like this when the balloon size is provided in the API.

Reply:
This is the memory size with balloon.  It should small than ram size.
Balloon can be used to decrease the memory usage of the guest.